arkouda_async_proxy_server/src/server.rs

```diff
@@ -0,0 +1,161 @@
+use std::env;
+
+use log::{info,debug};
+
+use tonic::{transport::Server, Request, Response, Status};
+
+use arkouda::arkouda_server::{Arkouda, ArkoudaServer};
+use arkouda::{ArkoudaRequest, ArkoudaResponse};
+
+use serde::{Deserialize, Serialize};
+use serde_json::{json};
+use serde_json;
+use tokio::task::{spawn,JoinHandle};
+
+use std::collections::HashMap;
+use std::sync::Arc;
+use tokio::sync::RwLock;
+
+pub fn print_type_of<T>(_: &T) {
+    println!("{}", std::any::type_name::<T>())
+}
```

Comment:
seems like this is prob left over from debugging but I might be wrong about that

Reply:
@pierce314159 yup, removed!